ChatRPG/Pages/UserCampaignOverview.razor

```diff
@@ -1,63 +1,75 @@
 @using ChatRPG.Data.Models
+@using ChatRPG.Services
 @using CampaignModel = ChatRPG.Data.Models.Campaign;
 
 <div class="container">
     @if (User is not null)
     {
+        <div class="dashboard-title-container">
+            <h1 class="title dashboard-title text-center">Dashboard</h1>
+        </div>
         <div class="row gap-3">
             <div class="col rounded custom-gray-container">
                 <div class="m-3">
-                    <h3>Your Campaigns</h3>
-                    <div class="vstack overflow-auto form-text-black" style="max-height: 400px;" id="scrollbar">
+                    <h3 class="text-center mb-4">Your Campaigns</h3>
+                    <div class="vstack overflow-auto form-text-black mb-1 scrollbar" style="max-height: 400px;" id="your-campaigns">
                         @foreach (CampaignModel campaign in Campaigns)
                         {
-                            <div class="card" style="width: 18rem;">
-                                <div class="card-body">
+                            <button class="card card-dim mb-2" @onclick='() => LaunchCampaign(campaign.Id)'>
+                                <div class="card-body col-12">
                                     <h5 class="card-title">@campaign.Title</h5>
-                                    <h6 class="card-subtitle mb-2 text-muted">@campaign.Characters.First(c => c.IsPlayer).Name</h6>
-                                    <p class="card-text text-truncate">@(campaign.CustomStartScenario ?? campaign.StartScenario?.Body ?? "No scenario")</p>
-                                    <p class="card-text">Started @campaign.StartedOn.ToShortDateString() @campaign.StartedOn.ToShortTimeString()</p>
-                                    <a href="/campaign/@campaign.Id" class="card-link">See more ></a>
+                                    <h6 class="card-subtitle mb-2 text-muted" style="font-size: 14px;">@campaign.Characters.First(c => c.IsPlayer).Name</h6>
+                                    <hr/>
+                                    <p class="card-text line-clamp text-start">@(campaign.CustomStartScenario ?? campaign.StartScenario?.Body ?? "No scenario")</p>
```

Comment:
Eftersom vi sløjfer relationen til `StartScenario`, og i stedet bare gemmer teksten i `CustomStartScenario`
```suggestion
                                    <p class="card-text line-clamp text-start">@(campaign.CustomStartScenario ?? "No scenario")</p>
```

Reply:
Inden jeg fixer denne her. Kan vi ikke bare kalde det StartScenario og så sløjfe CustomStartScenario? Det er mest navngivningen jeg tænker på. Custom er ikke nødvendig, hvis vi alligevel kun har én måde at gøre det på.

Reply:
Sagtens - du kan enten selv gøre det og tilføje en ny migration, ellers kan du bare lade den være CustomStartScenario og så fixer vi efter det her PR

Reply:
Og bare resolve den her kommentar når du har besluttet hvad du vil

Reply:
Jeg vil gerne prøve, men jeg vil egentlig gerne have du lige ser på det commit, for jeg har aldrig prøvet det før.

Reply:
Håber jeg har gjort det rigtigt. Du må gerne tjekke https://github.com/KarmaKamikaze/ChatRPG/pull/42/commits/303e6d8cf4e4a0697cd267a3cc8fe946db890cf9

Reply:
Vi prøver igen med https://github.com/KarmaKamikaze/ChatRPG/pull/42/commits/2db82b557381b572929d7067fb42277725612158

Reply:
Det er perfekt. Jeg lukker denne kommentar